juspay/hyperswitch
# feat(mandate_kv): add kv support for mandate

## Type of Change


- [ x] Enhancement

## Description

Mandate table is being written to db in payment flow, this change moves all db writes and updates to kv

### Additional Changes





## Motivation and Context

Mandate table is being written to db in payment flow, this change moves all db writes and updates to kv

## How did you test it?

sanity check on payment flow

test list:
1. create payment with mandate for merchant with kv enabled
2. check drainer for errors and verify in db if mandate entry is written

## Checklist
